suspicious_content.txt

```diff
@@ -1,44 +1,49 @@
- a secured file with you
- all parties completed
- all parties have completed
- all signers completed
- antivirus status
- but there was nobody who could sign
- clustered messages
- document has been completed
- download as excel
- download.as.excel
- dropbox file sent
- exceeded your
- file scanned by avast antivirus
- gift card 
- shared the following document
- help desk
- help-desk
- helpdesk
- how to resolve your email
- new documents was sent to you
- new documents were sent to you
- no emails reply: no emails
- one drive team
- our courier attempted
- password reset
- purchase order
- scan result: clean
- secured files link
- security alert
- share a file
- shared a private document
- shared a secure file
- suspension
- suspicious activity
- suspicious login
- to share some document
- unusual
- used docu.?box to
- verify your
- view click here
- view invoice in docxn
- will be deleted in a few hours
- will be deleted in few hours
- you have new documents
+a secured file with you
+all parties completed
+all parties have completed
+all signers completed
+antivirus status
+but there was nobody who could sign
+clustered messages
+document has been completed
+download as excel
+download.as.excel
+dropbox file sent
+exceeded your
+file scanned by avast antivirus
+gift card 
+shared the following document
+help desk
+help-desk
+helpdesk
+how to resolve your email
+incoming .?.?fax
+new documents was sent to you
+new documents were sent to you
+new sign.?in
```

Comment:
does this mean `[-\s]?`?

Reply:
i think so yeah. `sign-in` `sign in` `signin`. do you prefer that since it's more explicit?